cmd/tempo-cli/cmd-convert-block.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"os"
+
+	"github.com/grafana/tempo/tempodb/encoding/vparquet"
+	"github.com/segmentio/parquet-go"
+)
+
+type convertParquet struct {
+	In  string `arg:"" help:"The input parqruet file to read from"`
+	Out string `arg:"" help:"The output parquet file to write to"`
+}
+
+func (cmd *convertParquet) Run() error {
+	// open In
+	fIn, err := os.Open(cmd.In)
+	if err != nil {
+		return err
+	}
+	s, err := fIn.Stat()
+	if err != nil {
+		return err
+	}
+	pf, err := parquet.OpenFile(fIn, s.Size())
+	if err != nil {
+		return err
+	}
+
+	// open Out
+	fOut, err := os.Create(cmd.Out)
+	if err != nil {
+		return err
+	}
+	sch := parquet.SchemaOf(new(vparquet.Trace))
+	writer := parquet.NewWriter(fOut, sch)
+
+	conversion, err := parquet.Convert(sch, pf.Schema())
+	if err != nil {
+		return err
+	}
+
+	// copy a rowgroup at a time
```

Comment:
Would it make sense to add some output to command execution, possibly progress over `len(pf.RowGroups)` processed? (Percentage or 'x out of n processed' - based on some time interval).

Reply:
Thanks for the suggestion. Added some progress messages.